CMake/curl-config.cmake.in

```diff
@@ -23,7 +23,12 @@
 ###########################################################################
 @PACKAGE_INIT@
 
+option(CURL_USE_PKGCONFIG "Enable pkg-config to detect @PROJECT_NAME@ dependencies. Default: @CURL_USE_PKGCONFIG@" "@CURL_USE_PKGCONFIG@")
+
 include(CMakeFindDependencyMacro)
+set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR} ${CMAKE_MODULE_PATH})
```

Comment:
`CMAKE_MODULE_PATH` shall be saved and restored. The extra find modules must only be used during `find_package(CURL CONFIG)`. But they must not affect `find_package` after returning the calling scope.
~~~cmake
find_package(CURL CONFIG) # yours
find_package(Brotli)      # mine!
~~~